bb-new.html

```diff
@@ -0,0 +1,296 @@
+              <li class="nav-list-item"><a href="#">Developer Blog</a></li>
+              <li class="nav-list-item"><a href="#">Support Blog</a></li>
+            </ul>
+          </div>    
+        </div>
+        <div class="col">
+          <a href="#blog">Seach icon</a>
+        </div>
+        <div class="col">
+          <a href="#headphone">Headphone icon</a>  
+          <div class="dropwdown">
+            <ul>
+              <li class="nav-list-item"><a href="#">Register Products</a></li>
+              <li class="nav-list-item"><a href="#">Contact Sales</a></li>
+              <li class="nav-list-item"><a href="#">Contact Us</a></li>
+            </ul>
+          </div>    
+        </div>                 
+        </nav>                 
+            <div class="row">
+              <div class="col">
+                <h1>BlackBerry 2021 Threat Report Now Available</h1>
+                <p>Learn about the most significant cybersecurity events of 2020 so you can prepare for the year ahead</p>
+                <button>READ THE REPORT</button>
+              </div>
+            </div>
+          </header>         
+        </div>         
+      </div>
+      <div class="row">
+        <div class="col">
+          <h2>SECURITY SHOULD'T JUST BE SMART. IT SHOULD BE INTELLIGENT</h2>
+          <p>We’re known for secure handsets that changed the world, and how we work. Today, we’re reimagining protection, again, with artificial intelligence and machine learning. We make everything more secure—and you, more productive.</p>
+          <ul>
+            <li class="sec-list-items"><a href="#">CYBERSECURITY</a></li>
+            <li class="sec-list-items"><a href="#">CRITICAL EVENT MANAGEMENT</a></li>
+            <li class="sec-list-items"><a href="#">INTERNET OF THINGS (loT)</a></li>
+          </ul>
+        </div>
+        <div class="col">
+          <img src="/images/men-holding-laptop-on-baconey.jpg" alt="men holding laptop outside on a baconey">
+        </div>
+      </div>
+      <div class="row">
+        <div class="col">
+          <img src="/images/men-back-sitting-on-chair.jpg" alt="men back sitting on a chair facing a laptop">
+        </div>
+      </div>
+      <div class="row">
+        <div class="col">
+          <img src="/images/security-icon.jpg" alt="blackberry security icon">
+          <h2>WE’VE NEVER BEEN MORE CONNECTED.OR MORE VULNERABLE.</h2>
+          <p>Cybersecurity has failed to keep up, because it fails to look ahead. Our intelligent security pairs artificial intelligence with machine learning to proactively protect your system from cyberthreats. It’s time to protect, prevent and respond.</p>
+          <button syle="display: inline-block">CYBER SUITE</button>
+          <button style="display: inline block">SPARK SUITE</button>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col">
+          <img src="/images/security-icon-2.jpg" alt="blackberry security icon">
```

Comment:
This needs to be in its own column.
And set up so that it's to the right of the "Always-On.  Always Protected" content below